src/models/connectionProfile.ts

```diff
@@ -41,11 +41,11 @@ export class ConnectionProfile
     public savePassword: boolean;
     public emptyPasswordInput: boolean;
     public azureAuthType: AzureAuthType;
-    public declare azureAccountToken: string | undefined;
-    public declare expiresOn: number | undefined;
+    declare public azureAccountToken: string | undefined;
+    declare public expiresOn: number | undefined;
     public accountStore: AccountStore;
-    public declare accountId: string;
-    public declare tenantId: string;
+    declare public accountId: string;
+    declare public tenantId: string;
```

Comment:
Linter was complaining about this while merging main. I'm going to undo my change to the prettier dev dependency version while I was trying to get closing brackets to appear on the same line to see if I can get the changes in this file out of this PR.